### ISSUE TITLE
Feature request from user: HCPF calculation in `VolumetricsAnalysis`

### ISSUE THREAD

**Issue by asnyv** [equinor/webviz-subsurface#1099]
User wants to be able to plot/tabulate hydrocarbon pore fraction (HCPF = HCPV/bulk volume) based on the selections in the `VolumetricsAnalysis` plugin similar to what is done for e.g. water saturation (Sw).

Additional comment:
are there other properties we could calculate? obvious options are Sg and So in addition to the existing Sw.

**Comment by asnyv** [comment]
@tnatt thougts?

**Comment by tnatt** [comment]
I agree @asnyv, it could be nice to have these columns available, and it should be quite easy to implement. 
Another column that would be very good for qc and has been requested by users is to calculate facies fractions (if facies is present in the dataset). Per now only NTG is calculated, but for qc'ing that we model the volume fractions that has been set-up in rms we need to calculate fractions for the individual facies.
